Add a Javascript file

BryteLitty opened the pull request:
This is my first Microverse Coding project and the following are what I have done.

- Created a Github repo
- created a new branch with the name "add-javascript-file"
- set up linting
- created a .gitignore file to make sure files that are not relevant are ignored by git
- added project description in a README.md file
- added an HTML file
- added a CCS file
- added test.md file
- added test.md to .gitignore 
- opened a pull request

BryteLitty committed:
Add javascript file

franclobo commented on style.css:
```diff
@@ -1,3 +1,3 @@
-    background-color: purple;
-}
\ No newline at end of file
+.bg-color {
+  background-color: purple;
+}
```
- [ ] Kindly add the link to the CSS file in your `index.html` code to see the styles in the browser. 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/191385718-5d340d6f-1972-4c31-ba96-d3317426ec8b.png)

franclobo reviewed (changes requested):
Hi @BryteLitty, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please 🙏🏽 document your work with meaningful commits of each document created or updated, those are `index.html`, `styles.css`, and `test. md`.
![image](https://user-images.githubusercontent.com/58642949/191387863-f7359d6e-24f5-4ba2-b240-444faab16590.png)

- [x] Please 🙏🏽 remove the `homepage.html` file from the `.gitignore` file just the `test.md` and `node_modules/` should be in  the `.gitignore` file.
- [x] Neither an empty `index.js` file nor the `hint-report` folder is needed in this project, please 🙏🏽 remove them.
- [x] Kindly add your name in the Authors sections and please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

![image](https://user-images.githubusercontent.com/58642949/191387165-b0a9ac1e-7e24-475a-b52b-d8f7ace8f5bd.png)

- [x] Kindly add the LICENSE file and section. Please 🙏🏽  make sure that the link works as expected for a professional README file. 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/191386850-6930e601-b217-4cd7-bd91-0a50fbd934b8.png)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BryteLitty committed:
Create LICENSE file

BryteLitty committed:
link CSS to html

BryteLitty committed:
update after pull request

BryteLitty committed:
Merge branch 'add-javascript-file' of https://github.com/BryteLitty/hello-microverse into add-javascript-file

BryteLitty committed:
License section added to README.md

BryteLitty commented:
Hello @franclobo thank you for your feedback. I have done the following changes and waiting for review.

-  linked the css file to the html
- added LICENSE file to the project
- updated my .gitignore
- update the README.md file with License section

Reem-lab reviewed (approved):
# Hi @BryteLitty,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯
<img src="https://media1.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>


## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌Professional README ✅

## Optional suggestions
- [ OPTIONAL ] I think you install the backages in wrong way please remove the node_modules and package/json and package-lock.json files then install the packages again by following [these](https://github.com/microverseinc/linters-config/tree/master/html-css) steps if you scroll down you will find them in order

This three commands for installing the packages this is one time per project put there are two comands also to check the linters error for the HTML page and for the style page 😊💯🔥

> Step 1
![image](https://user-images.githubusercontent.com/58553711/191396366-6fd9cd04-debe-45cb-868b-61e5b873a4f4.png)

> Step 2
![image](https://user-images.githubusercontent.com/58553711/191396478-da31aede-8a2c-4ef7-9a7e-e28f608b74ef.png)


- [ OPTIONAL ] please remove unused files like `index.js` and `hint-report` since they are not necceary to add them 🔥💯😊
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BryteLitty committed:
corrections after review

frank1738 reviewed (approved):
Hi @BryteLitty , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

![YesYouAreGIF (2)](https://user-images.githubusercontent.com/50941074/172881298-64e98170-87c5-4075-8a77-71d65d991f55.gif) 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
